### PR TITLE
Add refs for all tables

### PR DESCRIPTION
**Pull request by SoongNoonien**
These are just the references for the character tables. The Greenfunction tables are still missing.

### CODE REVIEW

File: docs/src/refs.bib [hunk 1]
```diff
@@ -1,4 +1,158 @@
...
+  pages         = {584--614},
+  year          = {1984},
+  doi           = {10.1016/0021-8693(84)90084-X},
+  language      = {English},
+  zbmath        = {3857372}
+}
+
+@InCollection{CR74,
+  author        = {Chang, Bomshik and Ree, Rimhak},
+  title         = {The characters of {{\(G_ 2(q)\)}}},
```
Comment (fingolfin) [inline docs/src/refs.bib:18]: I think this won't render right, you need to use `$` for math mode (full LaTeX is not supported). 
```suggestion
  title         = {The characters of $G_2(q)$},
```

File: docs/src/refs.bib [hunk 2]
```diff
@@ -1,4 +1,158 @@
 
+@Article{BS84,
+  author        = {Beynon, W. M. and Spaltenstein, N.},
+  title         = {Green functions of finite {Chevalley} groups of type {{\(E_ n (n=6,7,8)\)}}},
```
Comment (fingolfin) [inline docs/src/refs.bib:4]:
```suggestion
  title         = {Green functions of finite {Chevalley} groups of type $E_n (n=6,7,8)$},
```

File: docs/src/refs.bib [hunk 3]
```diff
@@ -1,4 +1,158 @@
...
+  howpublished  = {Pure and {Applied} {Mathematics}. {A} {Wiley}-{Interscience} {Publication}. {Chichester}-{New} {York}
+                  etc.: {John} {Wiley} and {Sons}. {XII}, 544 p. {{\textsterling}} 42.50 (1985).},
+  year          = {1985},
+  language      = {English},
+  zbmath        = {3904833}
+}
+
+@Article{Cha68,
+  author        = {Chang, Bomshik},
+  title         = {The conjugate classes of {Chevalley} groups of type {{\((G_2)\)}}},
```
Comment (fingolfin) [inline docs/src/refs.bib:40]:
```suggestion
  title         = {The conjugate classes of {Chevalley} groups of type $G_2$},
```

File: docs/src/refs.bib [hunk 4]
```diff
@@ -1,4 +1,158 @@
...
+  pages         = {190--211},
+  year          = {1968},
+  doi           = {10.1016/0021-8693(68)90020-3},
+  language      = {English},
+  zbmath        = {3447168}
+}
+
+@Article{DM87,
+  author        = {Deriziotis, D. I. and Michler, G. O.},
+  title         = {Character table and blocks of finite simple triality groups{{\( ^ 3D_ 4(q)\)}}},
```
Comment (fingolfin) [inline docs/src/refs.bib:54]:
```suggestion
  title         = {Character table and blocks of finite simple triality groups ${}^3D_4(q)$},
```

File: docs/src/refs.bib [hunk 5]
```diff
@@ -13,6 +167,98 @@ @Article{GHLMP96
...
+
+@Article{Hot77,
+  author        = {Hotta, R., Springer, T.A.},
+  title         = {A Specialization Theorem for Certain Weyl Group Representations and an Application to the Green
+                  Polynomials of Unitary Groups.},
+  journal       = {Inventiones mathematicae},
+  volume        = {41},
+  pages         = {113--128},
+  year          = {1977},
+  url           = {http://eudml.org/doc/142488}
```
Comment (fingolfin) [inline docs/src/refs.bib:234]: add DOI etc. from https://zbmath.org/?q=an:0389.20037 ?

File: docs/src/refs.bib [hunk 6]
```diff
@@ -13,6 +167,98 @@ @Article{GHLMP96
...
+
+@Misc{Kaw85,
+  author        = {Kawanaka, N.},
+  title         = {Generalized {Gelfand}-{Graev} representations and {Ennola} duality},
+  zbl           = {0573.20038},
+  howpublished  = {Algebraic groups and related topics, {Proc}. {Symp}., {Kyoto} and {Nagoya}/{Jap}. 1983, {Adv}.
+                  {Stud}. {Pure} {Math}. 6, 175-206 (1985).},
+  year          = {1985},
+  language      = {English},
+  zbmath        = {3914571}
```
Comment (fingolfin) [inline docs/src/refs.bib:259]: DOI and generally better data from AMS MathSciNet:
```
@incollection {MR803335,
    AUTHOR = {Kawanaka, N.},
     TITLE = {Generalized {G}el\cprime fand-{G}raev representations and
              {E}nnola duality},
 BOOKTITLE = {Algebraic groups and related topics ({K}yoto/{N}agoya, 1983)},
    SERIES = {Adv. Stud. Pure Math.},
    VOLUME = {6},
     PAGES = {175--206},
 PUBLISHER = {North-Holland, Amsterdam},
      YEAR = {1985},
      ISBN = {0-444-87711-8},
   MRCLASS = {20G05},
  MRNUMBER = {803335},
MRREVIEWER = {Bhama\ Srinivasan},
       DOI = {10.2969/aspm/00610175},
       URL = {https://doi.org/10.2969/aspm/00610175},
}
```

File: docs/src/refs.bib [hunk 7]
```diff
@@ -26,6 +272,121 @@ @Article{Kem21
...
+  author        = {Lascoux, Alain and Sch{\"u}tzenberger, Marcel-Paul},
+  title         = {Sur une conjecture de {H}. {O}. {Foulkes}},
+  zbl           = {0374.20010},
+  journal       = {C. R. Acad. Sci., Paris, S{\'e}r. A},
+  fjournal      = {Comptes Rendus Hebdomadaires des S{\'e}ances de l'Acad{\'e}mie des Sciences, S{\'e}rie A},
+  volume        = {286},
+  pages         = {323--324},
+  year          = {1978},
+  language      = {French},
+  zbmath        = {3582368}
```
Comment (fingolfin) [inline docs/src/refs.bib:285]: please check if MathSciNet has DOI

File: docs/src/refs.bib [hunk 8]
```diff
@@ -37,3 +398,160 @@ @Book{OSCAR
...
+  title         = {The conjugacy classes of {Chevalley} groups of type {{\((F_4)\)}} over finite fields of
+                  characteristic 2},
+  zbl           = {0306.20013},
+  journal       = {J. Fac. Sci., Univ. Tokyo, Sect. I A},
+  fjournal      = {Journal of the Faculty of Science. Section I A},
+  volume        = {21},
+  pages         = {133--159},
+  year          = {1974},
+  language      = {English},
+  zbmath        = {3479126}
```
Comment (fingolfin) [inline docs/src/refs.bib:443]: Check for DOI at MathSciNet

File: docs/src/refs.bib [hunk 9]
```diff
@@ -37,3 +398,160 @@ @Book{OSCAR
...
+  pages         = {505--543},
+  year          = {1982},
+  doi           = {10.1080/00927878208822732},
+  language      = {English},
+  zbmath        = {3762296}
+}
+
+@Book{Spa82,
+  author        = {Spaltenstein, Nicolas},
+  title         = {Classes unipotentes et sous-groupes de {Borel}},
```
Comment (fingolfin) [inline docs/src/refs.bib:476]: DOI and maybe more at https://link.springer.com/book/10.1007/BFb0096302

File: docs/src/refs.bib (hunk 7 above)
Comment (SoongNoonien) [reply docs/src/refs.bib:285]: This is what MathSciNet has:
```@article {MR472993,
    AUTHOR = {Lascoux, Alain and Sch\"utzenberger, Marcel-Paul},
     TITLE = {Sur une conjecture de {H}. {O}. {F}oulkes},
   JOURNAL = {C. R. Acad. Sci. Paris S\'er. A-B},
  FJOURNAL = {Comptes Rendus Hebdomadaires des S\'eances de l'Acad\'emie des
              Sciences. S\'eries A et B},
    VOLUME = {286},
      YEAR = {1978},
    NUMBER = {7},
     PAGES = {A323--A324},
      ISSN = {0151-0509},
   MRCLASS = {20C30 (14M15)},
  MRNUMBER = {472993},
}
```

I don't know which one is better.

File: docs/src/refs.bib (hunk 8 above)
Comment (SoongNoonien) [reply docs/src/refs.bib:443]: Apparently also know doi:
```
@article {MR349863,
    AUTHOR = {Shinoda, Ken-ichi},
     TITLE = {The conjugacy classes of {C}hevalley groups of type
              {$(F\sb{4})$} over finite fields of characteristic {$2$}},
   JOURNAL = {J. Fac. Sci. Univ. Tokyo Sect. IA Math.},
  FJOURNAL = {Journal of the Faculty of Science. University of Tokyo.
              Section IA. Mathematics},
    VOLUME = {21},
      YEAR = {1974},
     PAGES = {133--159},
      ISSN = {0040-8980},
   MRCLASS = {20G40},
  MRNUMBER = {349863},
MRREVIEWER = {R.\ W.\ Carter},
}
```

File: docs/src/refs.bib (hunk 7 above)
Comment (fingolfin) [reply docs/src/refs.bib:285]: Either is fine -- I was mainly hoping for a DOI or URL, but there doesn't seem to be any, so you can leave it as it is.

File: docs/src/refs.bib (hunk 8 above)
Comment (fingolfin) [reply docs/src/refs.bib:447]: Indeed! Well it took me quite some effort but the DOI is 10.15083/00039769

File: docs/src/refs.bib [hunk 10]
```diff
@@ -1,4 +1,158 @@
...
+
+@Misc{Car85,
+  author        = {Carter, Roger W.},
+  title         = {Finite groups of {Lie} type. {Conjugacy} classes and complex characters},
+  zbl           = {0567.20023},
+  howpublished  = {Pure and {Applied} {Mathematics}. {A} {Wiley}-{Interscience} {Publication}. {Chichester}-{New} {York}
+                  etc.: {John} {Wiley} and {Sons}. {XII}, 544 p. £42.50 (1985).},
+  year          = {1985},
+  language      = {English},
+  zbmath        = {3904833}
```
Comment (fingolfin) [inline docs/src/refs.bib:35]: Better:
```
@book {MR794307,
    AUTHOR = {Carter, Roger W.},
     TITLE = {Finite groups of {L}ie type},
    SERIES = {Pure and Applied Mathematics (New York)},
      NOTE = {Conjugacy classes and complex characters,
              A Wiley-Interscience Publication},
 PUBLISHER = {John Wiley \& Sons, Inc., New York},
      YEAR = {1985},
     PAGES = {xii+544},
      ISBN = {0-471-90554-2},
   MRCLASS = {20G40 (20-02 20C15)},
  MRNUMBER = {794307},
MRREVIEWER = {David\ B.\ Surowski},
}
```
The first line of the "note" probably should be a `subtitle` (not sure if `DocumenterCitation` supports those)

File: docs/src/refs.bib [hunk 11]
```diff
@@ -1,4 +1,158 @@
...
+  author        = {Enomoto, Hikoe and Yamada, Hiromichi},
+  title         = {The characters of $G_2(2^n)$},
+  zbl           = {0614.20028},
+  journal       = {Jpn. J. Math., New Ser.},
+  fjournal      = {Japanese Journal of Mathematics. New Series},
+  volume        = {12},
+  pages         = {325--377},
+  year          = {1986},
+  language      = {English},
+  zbmath        = {3993856}
```
Comment (fingolfin) [inline docs/src/refs.bib:88]: better in mathscinet:
```
@article {MR914301,
    AUTHOR = {Enomoto, Hikoe and Yamada, Hiromichi},
     TITLE = {The characters of {$G_2(2^n)$}},
   JOURNAL = {Japan. J. Math. (N.S.)},
  FJOURNAL = {Japanese Journal of Mathematics. New Series},
    VOLUME = {12},
      YEAR = {1986},
    NUMBER = {2},
     PAGES = {325--377},
      ISSN = {0289-2316},
   MRCLASS = {20C15},
  MRNUMBER = {914301},
MRREVIEWER = {B.\ Chang},
       DOI = {10.4099/math1924.12.325},
       URL = {https://doi.org/10.4099/math1924.12.325},
}
```

File: docs/src/refs.bib [hunk 12]
```diff
@@ -1,4 +1,158 @@
...
+  author        = {Ennola, Veikko},
+  title         = {On the conjugacy classes of the finite unitary groups},
+  zbl           = {0105.02403},
+  journal       = {Ann. Acad. Sci. Fenn., Ser. A I},
+  fjournal      = {Annales Academiae Scientiarum Fennicae. Series A I},
+  volume        = {313},
+  pages         = {13},
+  year          = {1962},
+  language      = {English},
+  zbmath        = {3171183}
```
Comment (fingolfin) [inline docs/src/refs.bib:101]: no doi, check mathscinet?

Comment (fingolfin) [reply docs/src/refs.bib:101]: also for the others without doi

File: docs/src/refs.bib [hunk 13]
```diff
@@ -13,6 +167,105 @@ @Article{GHLMP96
...
+  pages         = {113--127},
+  year          = {1977},
+  doi           = {10.1007/BF01418371},
+  url           = {https://eudml.org/doc/142488},
+  language      = {English},
+  zbmath        = {3605124}
+}
+
+@Book{Hi90,
+  author        = {Hi{\ss}, Gerhard},
```
Comment (fingolfin) [inline docs/src/refs.bib:233]: we can and probably should use Unicode in stead of these
```suggestion
  author        = {Hiß, Gerhard},
```

File: docs/src/refs.bib [hunk 14]
```diff
@@ -26,6 +279,119 @@ @Article{Kem21
...
+
+@Book{Lub93,
+  author        = {L{\"u}beck, Frank},
+  title         = {Charaktertafeln f{\"u}r die {Gruppen} $\text{CSp}_6(q)$ mit ungeradem $q$ und $\text{Sp}_6(q)$ mit
+                  geradem $q$.},
+  zbl           = {0832.20022},
+  publisher     = {Heidelberg: Universit{\"a}t Heidelberg, Naturw.-Math. Gesamtfakult{\"a}t},
+  year          = {1993},
+  language      = {German},
+  zbmath        = {444262}
```
Comment (fingolfin) [inline docs/src/refs.bib:332]: Add link to https://www.math.rwth-aachen.de/~Frank.Luebeck/preprints/ or even directly to https://www.math.rwth-aachen.de/~Frank.Luebeck/preprints/CharTabCSp6.pdf ? Via the url field

File: docs/src/refs.bib [hunk 15]
```diff
@@ -1,4 +1,158 @@
...
+  volume        = {1896},
+  pages         = {985--1021},
+  year          = {1896},
+  language      = {German},
+  zbmath        = {2675572},
+  jfm           = {27.0092.01}
+}
+
 @Article{GHLMP96,
   author        = {Geck, Meinolf and Hiss, Gerhard and Lübeck, Frank and Malle, Gunter and Pfeiffer, Götz},
```
Comment (fingolfin) [inline docs/src/refs.bib:157]:
```suggestion
  author        = {Geck, Meinolf and Hiß, Gerhard and Lübeck, Frank and Malle, Gunter and Pfeiffer, Götz},
```

File: docs/src/refs.bib [hunk 16]
```diff
@@ -13,6 +167,105 @@ @Article{GHLMP96
   doi           = {10.1007/BF01190329}
 }
 
+@Article{GP92,
+  author        = {Geck, Meinolf and Pfeiffer, G{\"o}tz},
```
Comment (fingolfin) [inline docs/src/refs.bib:171]:
```suggestion
  author        = {Geck, Meinolf and Pfeiffer, Götz},
```

File: docs/src/refs.bib [hunk 17]
```diff
@@ -13,6 +167,105 @@ @Article{GHLMP96
...
+}
+
+@Book{Hi90,
+  author        = {Hi{\ss}, Gerhard},
+  title         = {Zerlegungszahlen endlicher {Gruppen} vom {Lie}-{Typ} in nicht-definierender {Charakteristik}.},
+  zbl           = {0832.20021},
+  publisher     = {Aachen: Techn. Hochsch. Aachen, Mathematisch-Naturwiss. Fakult{\"a}t},
+  year          = {1990},
+  language      = {German},
+  zbmath        = {51240}
```
Comment (fingolfin) [inline docs/src/refs.bib:239]: Could at least link to https://publications.rwth-aachen.de/record/821447

And then we could have a `note` or so which says "See also ..." and points to https://link.springer.com/chapter/10.1007/978-3-0348-8658-1_17 (and we should add that to the bib, too -- at the bottom of the page there is citation info, or perhaps zbmath or mathscinet has it, too)

Comment (SoongNoonien) [reply docs/src/refs.bib:240]: > And then we could have a note or so which says "See also ..." and points to https://link.springer.com/chapter/10.1007/978-3-0348-8658-1_17 (and we should add that to the bib, too -- at the bottom of the page there is citation info, or perhaps zbmath or mathscinet has it, too)

I'm sorry, I don't get it. Where could we add a note about what? Something like:
```suggestion
  zbmath        = {51240},
  note = {See also: https://link.springer.com/chapter/10.1007/978-3-0348-8658-1_17}
```

File: docs/src/refs.bib (hunk 10 above)
Comment (SoongNoonien) [reply docs/src/refs.bib]: It seems to be fine with `subtitle`.

File: docs/src/refs.bib (hunk 12 above)
Comment (SoongNoonien) [reply docs/src/refs.bib:102]: Maybe this `10.5186/aasfm.1962.313`? I couldn't find a doi for all papers.

**Review by fingolfin — APPROVED** [review]

**Pull request merged**